### PR TITLE
SP-8030: Get lightness range

### PR DESCRIPTION
**Pull request by michallowasrzechonek-silvair**

### CODE REVIEW

File: bluetooth_mesh/models/models.py [hunk 1]
```diff
@@ -1498,6 +1495,49 @@ async def request():
             request, retransmissions=retransmissions, send_interval=send_interval
         )
 
+    async def get_lightness_range(
```
Comment (bartoszflis-silvair) [inline bluetooth_mesh/models/models.py:1498]: How does `LightRangeCommand` know about this coro?

Comment (michallowasrzechonek-silvair) [reply bluetooth_mesh/models/models.py:1498]: `ModelGetCommandMixin`

Comment (michallowasrzechonek-silvair) [reply bluetooth_mesh/models/models.py:1498]: https://github.com/SilvairGit/python-bluetooth-mesh/blob/2bd0fbd6188e3fdd7bed4457795eb0f84e8abf40/bluetooth_mesh/apps/meshcli.py#L317-L335

Comment (bartoszflis-silvair) [reply bluetooth_mesh/models/models.py:1498]: Thanks.

**Review by bartoszflis-silvair — APPROVED** [review]

**Pull request merged**